MAUI/Scheduler/migration.md

```diff
@@ -22,6 +22,70 @@ To make the migration from the Xamarin [Xamarin SfSchedule](https://www.syncfusi
 <td>Syncfusion.Maui.Scheduler</td></tr>
 </table>
 
+## Initialize control
+
+To initialize the control, import the scheduler namespace and initialize SfScheduler as shown in the following code sample.
+
+<table>
+<tr>
+<th>Xamarin SfSchedule</th>
+<th>.NET MAUI SfScheduler</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:scheduler="clr-namespace:Syncfusion.Maui.Scheduler;assembly=Syncfusion.Maui.Scheduler">
```

Comment:
Add proper namepace for xamarin , seems used maui code

Reply:
Xamarin namespace added correctly,